src/compiler/program.ts

```diff
@@ -276,10 +276,10 @@ module ts {
 
         function processImportedModules(file: SourceFile, basePath: string) {

```

Comment:
Can we add a syntax error after parsing that the syntax is not allowed. just so that we can check this incrementally.